src/rgw/rgw_rest_swift.cc

```diff
@@ -777,11 +777,20 @@ int RGWGetObj_ObjStore_SWIFT::send_response_data(bufferlist& bl, off_t bl_ofs, o
 {
   string content_type;
 
-  if (sent_header)
+  if (sent_header) {
```

Comment:
In case of an error after the header was sent we won't send the error,
I think we should move it to after we check for an error.